decentraland/unity-renderer
# Chore: simple refactor to unify places requests

## What does this PR change?



Fix #4067 

...

## How to test the changes?



1. Launch the explorer https://play.decentraland.org/?explorer-branch=chore/places-refactor&ENABLE_FAVOURITE_PLACES=
2. Verify that the places sections are correctly displayed
3. Verify that adding and removing favorites is working correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
